ref(native): Add note about multi-process support (#8438)
getsentry/sentry-docs: src/platforms/native/configuration/backends/crashpad.mdx

It is advised to specify both the path to the `crashpad_handler` executable
as well as the path to the database directory manually.

> Multi-process use cases are also supported. Initialize Sentry individually
> in each process which spawns its own Crashpad handler either in a separate
> database path, or using a shared database path. Each run uses its individual
> run directory, and there is filesystem-level locking in place.

Reviewer: ```suggestion
You can also use Crashpad for multi-process use cases. To do this, initialize Sentry individually
in each process that spawns its own Crashpad handler, either in a separate
database path or using a shared database path. Each run will use its individual
run directory, and there is filesystem-level locking in place.
```

Reviewer: I found this original sentence to be a little confusing, so hopefully this reword makes it more clear and accurately reflects what you meant!

Author: Thanks for the suggestion! I adapted it a tiny bit, wdyt?

Reviewer: This looks awesome, thanks for taking the time to make sure this is really clear!
